### PR TITLE
Pull request for Content assist (issue #2)

### PR DESCRIPTION
**Pull request by gokcekrky**

### CODE REVIEW

File: de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/contentassist/CryptSLSprayProposalProvider.java [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+		});
+
+	}
+	@Override
+	public void completeAggregateExpression_Value(EObject model, Assignment assignment, ContentAssistContext context, ICompletionProposalAcceptor acceptor) {
+		if(context.getCurrentNode().hasPreviousSibling()) {
+			if(context.getCurrentNode().getPreviousSibling().hasPreviousSibling()) { 
+				if (context.getCurrentNode().getPreviousSibling().getPreviousSibling().getText().equals("CONSTRAINTS")) {
+						completeAggregateProposal(context,acceptor);
+				}else {
```
Comment (kruegers) [inline de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/contentassist/CryptSLSprayProposalProvider.java]: It looks like the call in line 84 is called in all cases when line 82 is not reached. If so, please collapse all the else branches into one statement.

File: de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/contentassist/CryptSLSprayProposalProvider.java [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+					if(contentAssistContext.getCurrentNode().getPreviousSibling().getText().length()<=0) {
+						completeAggregateProposal(contentAssistContext,acceptor);
+						completeConstraintProposal(contentAssistContext,acceptor);
+					}else if(contentAssistContext.getCurrentNode().getPreviousSibling().getText().contains(";")) {
+						String text = contentAssistContext.getCurrentNode().getPreviousSibling().getText();
+						if(text.charAt(text.length()-1) == ';') {
+							completeAggregateProposal(contentAssistContext,acceptor);
+							completeConstraintProposal(contentAssistContext,acceptor);
+						}
+						else{
```
Comment (kruegers) [inline de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/contentassist/CryptSLSprayProposalProvider.java]: Same.

File: de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/utils/ClassLoader.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+							return c;
+						} catch (ClassNotFoundException e1) {
+							// TODO Auto-generated catch block
+							System.out.println(classname + "is not found");
+						}
+		            	 found = true;
+		            }
+				}
+			} catch (IOException e1) {
+				// TODO Auto-generated catch block
```
Comment (kruegers) [inline de.darmstadt.tu.crossing.CryptSL.ui/src/de/darmstadt/tu/crossing/ui/utils/ClassLoader.java]: No todos.

**Comment by gokcekrky** [comment]
Hi @kruegers,
I made requested changes. Can you please have a look.

**Pull request merged**